sb_arch_opt/algo/segomoe_interface/api.py

```diff
@@ -23,5 +23,4 @@
 SOFTWARE.
 """
 from sb_arch_opt.algo.segomoe_interface.algo import *
-
-__all__ = ['HAS_SEGOMOE', 'SEGOMOEInterface']
+__all__ = ['HAS_SEGOMOE','HAS_SMT' ,'SEGOMOEInterface']
```

Comment:
Replace with `, 'HAS_SMT',` (spaces).

Also check this at other places in the code!